core/iwasm/common/wasm_runtime_common.h

```diff
@@ -853,7 +853,7 @@ wasm_runtime_set_wasi_args_ex(WASMModuleCommon *module, const char *dir_list[],
                               uint32 dir_count, const char *map_dir_list[],
                               uint32 map_dir_count, const char *env_list[],
                               uint32 env_count, char *argv[], int argc,
-                              int stdinfd, int stdoutfd, int stderrfd);
+                              int64 stdinfd, int64 stdoutfd, int64 stderrfd);
```

Comment:
why don't we use `os_raw_file_handle` just like we do in `wasm_runtime_init_wasi`?

Reply:
`wasm_runtime_init_wasi` isn't part of the external API (i.e. declared in `wasm_export.h`) so we're free to use any internal types.